src/Sulu/Bundle/WebsocketBundle/Connection/DummyConnection.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/*
+ * This file is part of the Sulu CMF.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+namespace Sulu\Bundle\WebsocketBundle\Connection;
+
+use Ratchet\ConnectionInterface;
+
+/**
+ * Dummy connection for fallback
+ */
+class DummyConnection implements ConnectionInterface
```

Comment:
i needed a dummy connection to use the same interface as ratchet websockets. the name if perhaps not best choosen i will rename it to fallback connection!